src/Illuminate/Database/Schema/MySqlSchemaState.php

```diff
@@ -87,7 +87,7 @@ protected function baseDumpCommand()
     {
         $command = 'mysqldump '.$this->connectionString().' --no-tablespaces --skip-add-locks --skip-comments --skip-set-charset --tz-utc --column-statistics=0';
 
-        if (! $this->connection->isMaria()) {
+        if ($this->connection->is('mariadb')) {
```

Comment:
You accidentally reversed this condition.

Reply:
Thanks!